test/testPlotWidget.cc

```
@@ -86,5 +87,447 @@ namespace minsky
+      box.x0 = 300;
+      box.y0 = 200;
+      box.x1 = 0;
+      box.y1 = 0;
+      
+      resize(box);
+      
+      EXPECT_EQ(300.0f, iWidth());
+      EXPECT_EQ(200.0f, iHeight());
+    }
+
+    TEST_F(PlotWidgetTest, numLinesWithPorts)
```

Clang (14.0.6)

[error] all parameters should be named in a function

(readability-named-parameter,-warnings-as-errors)